packages/addons/service/mpd/source/resources/language/English/strings.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```

Comment:
This is maybe an opportunity replace this with a strings.po file with appropriate label ids, see http://kodi.wiki/view/Language_support